src/main/cpp/TrajMath.cpp

```diff
@@ -0,0 +1,56 @@
+#include "TrajMath.h"
+
+double t34::TrajMath::GetFiringAngle()
+{
+    //                   ____________________
+    //          ⎛  v² - √ v⁴ - g(gx² + 2v²y)   ⎞
+    // θ = atan ⎜ ———————————————————————————— ⎟
+    //          ⎝               gx             ⎠
+    //
+    // Source:
+    //   Solving Ballistic Trajectories <https://www.forrestthewoods.com/blog/solving_ballistic_trajectories/>
+    //   See section “Firing Angle to Hit Stationary Target.”
+
+    const auto v² = m_note_max_velocity_mps * m_note_max_velocity_mps;
+    const auto v⁴ = v² * v²;
+    const auto x = m_target_distance_meters;
+    const auto x² = x * x;
+    const auto y = m_target_height_meters;
+    const auto g = 9.80665; // gravity
+
+    const auto gx² = g * x²;
+    const auto v² * y = v²y
+    const auto gx = g * x;
+
+    const auto numerator = v² - sqrt(v⁴ - (g * (gx² + (2  * v²y))));
+    const auto denominator = gx;
+    const auto θ = atan(numerator / denominator);
+
+    return RAD_TO_DEG(θ);
```

Comment:
Because C++ can handle an expanded Unicode character set in its variable names, it is possible to write code that looks a lot more like the equation it implements. That way, it is a little bit easier to see just how accurately the code implements the source equation.

Have a look at this, and let me know what you think. If you don't care for it, we can revert it.

Reply:
I like it!